rflink/protocol.py

```diff
@@ -300,29 +349,63 @@ def handle_event(self, event):
 class RepeaterProtocol(RflinkProtocol):
     """Repeat switch commands received."""
 
-    def handle_event(self, packet):
+    def handle_event(self, packet: PacketType) -> None:
         """Handle incoming packet from rflink gateway."""
         if packet.get("command"):
             task = self.send_command_ack(packet["id"], packet["command"])
             self.loop.create_task(task)
 
 
+@overload
+def create_rflink_connection(
+    port: int,
+    host: str,
```

Comment:
Can't we just make the `port` and `host` variables `Union` types? This would be more clear to read (and spot differences) than the current suggestion of 3 times the entire function declaration.

Reply:
I don't think so, because doing so would open up argument combinations that don't actually work and that will be flagged by mypy.